app/react/I18N/components/specs/__snapshots__/I18NMenu.spec.js.snap

```diff
@@ -14,17 +12,25 @@ exports[`I18NMenu when there is only one language should only render the inline
       ]
     }
   >
-    <button
-      aria-label="Add/edit translations"
-      className="menuNav-btn btn btn-default"
-      onClick={[Function]}
-      type="button"
+    <label
+      className="translation-switch"
     >
-      <Connect(Icon)
-        icon="language"
-        size="lg"
+      <input
+        aria-label="Add/edit translations"
+        checked={false}
+        className="translation-switch-input"
+        onChange={[Function]}
+        type="checkbox"
       />
-    </button>
+      <span
+        className="translation-switch-slider"
+      />
+    </label>
   </Connect(NeedAuthorization)>
-</ul>
+  <ul
+    aria-label="Languages"
+    className="menuNav-I18NMenu"
+    role="navigation"
+  />
```

Comment:
Are we really expecting an empty `ul` element?  This means that the fixtures are not passing any languages to the component.